locales/tam.json

```diff
@@ -90,16 +98,54 @@
   "BiometricScreen": {
     "unlock": "கைரேகை மூலம் திற"
   },
-  "HistoryTab": {
-    "noHistory": "இன்னும் வரலாறு கிடைக்கவில்லை",
-    "downloaded": "பதிவிறக்கம்",
-    "shared": "பகிரப்பட்டது",
-    "received": "பெறப்பட்டது",
-    "deleted": "நீக்கப்பட்டது"
+  "HistoryScreen": {
+    "noHistory": "இதுவரை வரலாறு இல்லை",
+    "downloaded": "கிடைத்தது",
+    "shared": "பகிர்ந்து கொண்டார்",
+    "received": "பெற்றது",
+    "deleted": "அகற்றப்பட்டது"
+  },
+  "SettingScreen": {
+    "header": "அமைப்புகள்",
+    "injiAsVerifierApp": "சரிபார்ப்பு பயன்பாடாக இன்ஜி",
+    "basicSettings": "அடிப்படை அமைப்புகள்",
+    "bioUnlock": "பயோமெட்ரிக்ஸ் மூலம் திறக்கவும்",
+    "language": "மொழி",
+    "aboutInji": "இன்ஜி பற்றி",
+    "injiTourGuide": "இன்ஜி சுற்றுலா வழிகாட்டி",
+    "logout": "வெளியேறு"
+  },
+  "AboutInji": {
+    "aboutInji": "இன்ஜி பற்றி",
+    "header": "இன்ஜி பற்றி",
+    "aboutDetailes": "இன்ஜி என்பது ஒரு மொபைல் பயன்பாடாகும், இது நற்சான்றிதழ்களைச் சேமிக்க டிஜிட்டல் பணப்பையாகப் பயன்படுத்தப்படலாம். எந்த இடத்திலும் நேரத்திலும், ஆஃப்லைன் மற்றும் ஆன்லைன் முறைகள் இரண்டிலும் அடையாளச் சரிபார்ப்பை இது அனுமதிக்கிறது.",
+    "forMoreDetailes": "மேலும் விவரங்களுக்கு",
+    "clickHere": "இங்கே கிளிக் செய்யவும்",
+    "version": "பதிப்பு: 0.4.1",
```

Comment:
we are supposed to get the version number dynamically. it should not be a static value . please refer AppMetaData.tsx for the implementation

Reply:
I fixed that now the version number would get dynamic.